src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -247,12 +259,22 @@ export default class PhotoField extends React.Component {
   }
 
   cropImage = () => {
-    const cropResult = this.refs.cropper.getCroppedCanvas().toDataURL();
-    this.setState({ cropResult });
+    const cropResult = {};
+    this.refs.cropper.getCroppedCanvas().toBlob(blob => {
+      const file = blob;
+      file.lastModifiedDate = new Date();
+      file.name = this.state.fileName;
+      cropResult.file = file;
+      cropResult.src = this.refs.cropper.getCroppedCanvas().toDataURL(blob.type);
```

Comment:
This is necessary to prevent the image from being resize per known issue of cropperjs library (as documented on their README).